easybuild/easyconfigs/w/Wannier90/Wannier90-2.0.1-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'MakeCp'
+
+name = 'Wannier90'
+version = '2.0.1'
+
+homepage = 'http://www.wannier.org'
+description = """A tool for obtaining maximally-localised Wannier functions"""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.wannier.org/code']
+
+# ugly hack
+prebuildopts = 'rm make.sys; touch make.sys; '
```

Comment:
OK, it's possible they're not taking care of making sure that the options specified on the command line trickle down properly... I'm not sure how to fix that easily.

How about patching the `include` line from the `src/Makefile` instead, either with an actual patch file (since it's likely the patch will keep working for future versions)?